src/encoded/tests/data/inserts/lab.json

```diff
@@ -6,6 +6,7 @@
         "address2": "10 Schattuck Street",
         "address1": "Biomedical Bioinfomatics",
         "city": "Boston",
+        "pi": "986b362f-4eb6-4a9c-8173-3ab267227e3d",
```

Comment:
Both this "4DN member" user and the "Test-4DN" lab have status `deleted` on the data portal. Is this OK?

Reply:
I think this is just a leftover of when some inserts were accidentally loaded into the portal.  I don't think it should be a problem.

Reply:
Looks good!